dnf5/commands/clean/clean.cpp

```diff
@@ -168,7 +170,14 @@ void CleanCommand::run() {
     }
 
     if (ec) {
-        throw std::runtime_error(fmt::format("Cannot iterate the cache directory: \"{}\"", cachedir.string()));
+        if (ec.value() == ENOENT) {
+            std::cout << libdnf5::utils::sformat(
+                             _("Cache directory \"{}\" does not exist. Nothing to clean."), cachedir.string())
+                      << std::endl;
+            return;
+        }
+        throw std::runtime_error(libdnf5::utils::sformat(
+            _("Cannot iterate the cache directory: \"{}\": {}"), cachedir.string(), ec.message()));
```

Comment:
Just nitpicking - how about removing the first colon from the message: `_("Cannot iterate the cache directory \"{}\": {}")`. It seems redundant to me.

Reply:
Sure.

Reply:
I changed the message.